docs/osquery/osquery.asciidoc

```diff
@@ -51,111 +48,167 @@ and you'll get suggestions for agents by name, ID, platform, and policy.
 or to the drag-and-drop *Lens* editor to create visualizations.
 . To view more information about the request, such as failures, open the *Status* tab.
-. To optionally save the query for future use, click *Save for later* and define the ID,
-description, and other
-<<osquery-manage-query,details>>.
+. To save the query for future use, click *Save for later* and define the ID,
+description, and other <<osquery-manage-query,details>>. 
 
-To view a history of the past queries you have run, open the *Live queries history*.
+[float]
+[[osquery-view-history]]
+===  View or rerun previous live queries
 
-* To replay a query, click image:images/play-icon.png[Right-pointing triangle].
+From the *Live queries history* section on the *Live queries* tab:
 
-* To view the query <<osquery-results,results>> and <<osquery-status,status>>,
-click image:images/table-icon.png[Table icon].
+* Click image:images/play-icon.png[Right-pointing triangle] to replay a query.
+
+* Click image:images/table-icon.png[Table icon] to view the query <<osquery-results,results>> and <<osquery-status,status>>.
 +
 [role="screenshot"]
 image::images/live-query-check-results.png[Results of OSquery]
 
 
 [float]
 [[osquery-schedule-query]]
-== Schedule queries
+== Schedule queries with packs
+
+Create packs to organize sets of queries. For example, you might create a pack that checks
+for IT compliance-type issues, and another pack that monitors for evidence of malware. 
+You can schedule packs to run for one or more agent policies. When scheduled, queries in the pack are run at the set intervals for all agents in those policies. Scheduling packs is optional. 
+
+. Open the **Packs** tab.
+
+. Click **Add pack** to create a new pack, or click the name of an existing pack, then **Edit** to add queries to an existing pack.
+
+. Provide the following fields:
+
+* The name of the pack.
 
-Group and schedule queries to run on a specified interval, in seconds.
-For example, you might create a group that checks
-for IT compliance-type issues, and
-another group that monitors for evidence of malware.
+* A short description of the pack.
 
-. Open the **Scheduled query groups** tab.
+* The agent policies where this pack should run. If no agent policies are set, then the pack is not scheduled.
 
-. Click a group name to view the details.
+. Add queries to schedule:
+
+* To add a query to the pack, click *Add query*, and then either add a saved query or enter a new query.
+Each query must include a unique query ID and the interval at which it should run. 
+Optionally, set the minimum Osquery version and platform,
+or <<osquery-map-fields,map ECS fields>>. When you add a saved query to a pack, this adds a copy of the query. A connection is not maintained between saved queries and packs.
+
+* To upload queries from a .conf query pack, drag the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
```

Comment:
should conf be in a different font: `.conf`?

Reply:
agreed - updated